LLama/LLamaContext.cs

```diff
@@ -24,6 +24,7 @@ public sealed class LLamaContext
         : IDisposable
     {
         private readonly ILogger? _logger;
+        private readonly State _emptyState;
```

Comment:
This empty state doesn't seem to be used anywhere?

Reply:
Right, forgot to clean it up after I switched to llama_kv_cache_clear, instead of setting the empty state.